src/refactor_nrepl/core.clj

```diff
@@ -65,18 +65,28 @@
   Should generally always be true, except in a few refactor-nrepl's unit tests."
   true)
 
+(def ^:private resources-dir-pattern
+  (re-pattern (str "resources" File/separator "?$")))
+
+(def ^:private target-dir-pattern
+  (re-pattern (str "target" File/separator "?$")))
+
+(defn irrelevant-dir?
+  "Does `f` look like a directory without files that should be analyzed/refactored?"
+  [^File f]
+  (let [s (-> f .toString)]
```

Comment:
Why not just `str`? It calls `.toString` anyways.

Reply:
b/c anecdotically it's more performant, and more importantly it signals that I want to invoke File#toString, not an arbitrary string representation (like `pr-str`)